model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -61,7 +61,14 @@ data class ResolvedLicenseInfo(
      * All copyright findings that could not be matched to a license finding, mapped to the [Provenance] where they were
      * detected.
      */
-    val unmatchedCopyrights: Map<Provenance, Set<CopyrightFinding>>
+    val unmatchedCopyrights: Map<Provenance, Set<CopyrightFinding>>,
+
+    /**
+     * The final concluded license of the package (or project).
+     * This entails a possible application of license choices.
+     * This is required because possible SPDX operands and their belonging source are lost in [licenses].
+     */
+    val finalConcludedLicense: SpdxExpression?
```

Comment:
Isn't this what we were planning to call `effectiveLicense`, @mnonnenmacher?

Reply:
Yes and no, we actually need two things. One for the aggregation of all license expressions from all sources (not only concluded license) into a single expression before any choice is applied. This could be called `resolvedLicense`  and should support filtering by `LicenseView` for consistency with the rest of the API. And one with applied license choices which could be the `effectiveLicense`.

Reply:
This again reminds me of the license handling docs that I started some time ago. Maybe this is a good point to start a discussion and contribute docs to https://github.com/oss-review-toolkit/ort/pull/3623 first.

Reply:
> One for the aggregation of all license expressions from all sources (not only concluded license) into a single expression before any choice is applied. This could be called resolvedLicense and should support filtering by LicenseView for consistency with the rest of the API.

At the moment this information is already available in `val licenses: List<ResolvedLicense>`.
While applying a license choice will remove/filter this information from this list at a later point, this is currently not the case.
Hence I'd rather add it in a future change that actually integrates the applying of the license choices.

> And one with applied license choices which could be the `effectiveLicense`.

The effective license would not be the product of the applied choice on only the `concludedLicense` if I understand correctly.

Reply:
I'm fine with adding properties on demand, but I think I don't understand the reason for adding this property now. Wouldn't it be better to add it in the PR that implements the application of license choices?

Reply:
As we agree with you, we removed the last commit that contained this change.